kernel/config-5.11.x-x86_64

```diff
@@ -421,8 +421,7 @@ CONFIG_X86_INTEL_TSX_MODE_OFF=y
 # CONFIG_X86_INTEL_TSX_MODE_AUTO is not set
 # CONFIG_X86_SGX is not set
 CONFIG_EFI=y
-CONFIG_EFI_STUB=y
-CONFIG_EFI_MIXED=y
+# CONFIG_EFI_STUB is not set
```

Comment:
This looks a bit odd to be disabled here and elsewhere. I can't remember but I think we may have used this on Windows.

Reply:
@fredericdalleau was this an intended change or an accident?

Reply:
Mmm weird, not something expected. I think I ran `make oldconfig` let me double check

Reply:
So it turns out this has been introduced by cross generating the x86 configs files from arm64.
The reverse also true, if I cross generate arm64 configs from x86 I get a different output for the arms configs.
I'm reworking it, i'll update soon.

Reply:
I've rebased this branch on @fredericdalleau 's latest, rebuilt the kernels (with `make push` on amd64 and arm64) and updated the shas for testing.

Reply:
OK, the CI is green for the rebuilt kernels 😅 . WDYT @rn?